includes/api/class-edd-api.php

```diff
@@ -304,33 +416,41 @@ private function invalid_key() {
 	 * @return void
 	 */
 	public function process_query() {
+
 		global $wp_query;
 
 		// Check for edd-api var. Get out if not present
-		if ( ! isset( $wp_query->query_vars['edd-api'] ) )
+		if ( ! isset( $wp_query->query_vars['edd-api'] ) ) {
```

Comment:
could be empty for slighly better performance if there's a set ?edd-api= where the = is empty string that would cut it off early before running all the validity checks